editors/code/package.json

```diff
@@ -2178,4 +2178,4 @@
             }
         ]
     }
-}
+}
```

Comment:
Nit: can you leave this untouched?

Reply:
whoops, sorry about that—i think my editor made this change.